code/modules/mob/living/carbon/human/life.dm

```diff
@@ -844,8 +844,7 @@
 		mind.changeling.regenerate()
 
 /mob/living/carbon/human/proc/handle_shock()
-	if(status_flags & GODMODE)	return 0	//godmode
-	if(!can_feel_pain())
+	if(!can_feel_pain() || status_flags & GODMODE)
```

Comment:
`(status_flags & GODMODE)` to avoid order of operations ambiguity.

Reply:
true